servers/lumen/app/Diagnosis.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+use Ramsey\Uuid\Uuid;
+
+class Diagnosis extends Model
+{
+    protected $connection = 'diagnosisDB';
+    protected $table = 'assignedDiagnosis';
+    public $timestamps = false;
+    protected $primaryKey = 'serverSideRowUuid';
+    protected $keyType = 'string';
+    public $incrementing = false;
+
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = [
+        'ptUuid', 'masterDiagnosisId', 'discontinueNote', 'recordChangedByUuid', 'recordChangedFromIPAddress',
```

Comment:
Field name should be 'deletedNote' instead of 'discontinueNote'

Reply:
Rename